ansible/roles/dev/files/bashrc

```diff
@@ -53,46 +53,8 @@ preset() {
 # intentionally undocumented, only list pclean in help for this
 
 pclean() {
-    pstop;
-
-    pushd ~/devel/
-    # Remove and unlink all development files in reverse order (uninstall pulp last)
-    for (( idx=${#REPOS[@]} ; idx>0 ; idx-- )) ; do
-        repo="${REPOS[idx-1]}"
-        if [ -d $repo ]; then
-            pushd $repo
-            sudo ./pulp-dev.py -U
-            popd
-        fi
-    done
-
-    # Destroy mongo DB and clean up pulp-related files
-    mongo pulp_database ~/drop_database.js;
```

Comment:
I think this is the only use of this file. Could you also remove the file and the ansible copy task?